docs/source/config/server.rst

```diff
@@ -0,0 +1,214 @@
+~~~~~~~~~~~~~~~~
+Full path to SQLite database file, used for storing users and sessions data.
+
+Default value is ``/tmp/mxcube-user.db``.
+
+``ALLOWED_CORS_ORIGINS``
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+List of origins that are allowed to connect to this server.
+This list is passed via ``cors_allowed_origins`` parameter to the `SocketIO <https://flask-socketio.readthedocs.io/en/latest/api.html#flask_socketio.SocketIO>`_ class constructor.
+Note that if this list is too restrictive, the front-end will fail to create a SocketIO connection to the back-end.
+
+``CERT``
+~~~~~~~~
+
+Used to enable SSL and set the type of host certificate.
+The following values are supported:
+
++--------+------------------------------------------------+
+| NONE   | SSL disabled                                   |
++--------+------------------------------------------------+
+| ADHOC  | SSL enabled, autogenerated certificate is used |
++--------+------------------------------------------------+
+| SIGNED | SSL enabled, signed certificate is used        |
++--------+------------------------------------------------+
+
+Default value is ``NONE``.
+
+``CERT_PEM``
+~~~~~~~~~~~~
+
+Path of the certificate file to use for SSL connections.
+
+``CERT_KEY``
+~~~~~~~~~~~~
+
+Path of the private key file to use for SSL connections.
+
+mxcube
+------
+
+The mxcube section allows you to customize miscellaneous aspects of the MxCube application.
+It have settings for the on-axis video stream, user account management, experiment mode.
+The section has the following syntax:
+
+.. code-block:: yaml
+
+    mxcube:
+      mode: <experiment mode>
+      USE_EXTERNAL_STREAMER: <boolean>
+      VIDEO_FORMAT: <format name>
+      VIDEO_STREAM_URL: <stream URL>
+      VIDEO_STREAM_PORT: <stream port>
+      usermanager:
+        class: <class name>
+        inhouse_is_staff: <boolean>
+        users:
+          - username: <user name>
+            role: <role name>
+          ...
+
+The ``mxcube`` section may contain keys as described below.
+
+``mode``
+~~~~~~~~
+
+Set the experiment mode for the application.
+Different modes tailor the presented UI to be suitable for a specific type of MX experiments.
+The following values are supported:
+
++--------------+--------------------------------------+
+| mode         | experiment type                      |
++==============+======================================+
+| OSC          | standard oscillation data collection |
++--------------+--------------------------------------+
+| SSX-CHIP     | SSX fixed target experiments         |
++--------------+--------------------------------------+
+| SSX-INJECTOR | SSX injector experiments             |
++--------------+--------------------------------------+
+
+Default mode is ``OSC``.
+
+``USE_EXTERNAL_STREAMER``
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Use an external process to stream the sample view video.
+When enabled, MxCube will invoke an appropriate hook on the camera hardware object to start the streamer.
```

Comment:
```suggestion
When enabled, MXCuBE will invoke an appropriate hook on the camera hardware object to start the streamer.
```

Reply:
Fun fact: there is a substitution `|project|` pre-configured in the `rst_prolog`:

https://github.com/mxcube/mxcubeweb/blob/ec1048b3d8fbf9360e6000a3b1f6264f26fa390f/docs/conf.py#L42-L44

So in principle we could write this:
```suggestion
When enabled, |project| will invoke an appropriate hook on the camera hardware object to start the streamer.
```
... and it should show up as `MXCuBE-Web` in the resulting output (HTML).

Now I am think it might make sense to add it as `|mxcube|`. 

I have not really investigated if/how we could do this for Markdown documents.